tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -376,6 +374,81 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_connection_wipe(conn));
         }
 
+        /* Override auto-chosen defaults with only RSA cert default. ECDSA still loaded, but not default. */
+        EXPECT_SUCCESS(s2n_config_set_cert_chain_and_key_defaults(server_config, &rsa_cert, 1));
```

Comment:
Can we move this into the beginning or end of a test block so we don't accidentally change the order? Preferably associate with whichever test scenario is relying on this state?

Reply:
I'll just clear the config and set exactly what I want, so that it is super clear. Does that sound reasonable?